apps/src/studio/blocks.js

```diff
@@ -2262,10 +2262,14 @@ exports.install = function (blockly, blockInstallOptions) {
       }
       if (options.restrictedDialog) {
         var functionArray = [];
-        var numRestrictedSayChoices = 59;
+        var numRestrictedSayChoices = 60;
         for (var i = 0; i < numRestrictedSayChoices; i++) {
+          var sayOption = msg[`saySpriteChoices_${i}`];
+          if (!sayOption) {
+            continue;
```

Comment:
Yes, but I think this is unnecessary. It's useful locally, but the staging build will just copy the English string to the other locales once I merge.